TriangleTestNG/test/java/TriangleTest.java

```diff
@@ -0,0 +1,27 @@
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import triangle.Triangle;
+
+import static org.testng.Assert.assertEquals;
+
+public class TriangleTest {
+
+    @DataProvider(name = "Detect triangle provider")
+    public Object[][] positiveKindTriangle() {
+        return new Object[][]{
+                {"equilateral", 100, 100, 100},
+                {"equilateral", Double.MAX_VALUE, Double.MAX_VALUE, Double.MAX_VALUE},
+                {"equilateral", Double.MIN_VALUE, Double.MIN_VALUE, Double.MIN_VALUE},
+                {"isosceles", 8, 8, 6},
+                {"isosceles", Double.MAX_VALUE, 20, Double.MAX_VALUE},

```

Comment:
you must also check variants:
 20, Double.MAX_VALUE, Double.MAX_VALUE
Double.MAX_VALUE, Double.MAX_VALUE,  20
+ all combinations with Double.MIN_VALUE